frontend/src/pages/ActivityReport/Pages/Review/SomeGoalsHaveNoPromptResponse.js

```diff
@@ -22,6 +62,10 @@ const SomeGoalsHaveNoPromptResponse = ({
     }
   }
 
+  const onClickRefresh = async () => {
```

Comment:
Might have missed it but do we have a test for this click?

Reply:
yeah, its in the test file with the same name